wp-content/themes/pub/wporg-learn-2024/src/style/_tag.scss

```diff
@@ -0,0 +1,14 @@
+.wp-block-post-terms.is-style-tag {
+	a {
```

Comment:
When there's only one nested style, I usually try to keep them on the same line so it makes searching within the code easier if you copy/paste from the dev tools.

You can search for `.wp-block-post-terms.is-style-tag a`.

Incredibly minor.